libnavigation-ui/src/test/java/com/mapbox/navigation/ui/route/MapRouteLineTest.kt

```diff
@@ -367,31 +367,92 @@ class MapRouteLineTest {
     }
 
     @Test
-    fun getBelowLayer() {
+    fun getBelowLayerWithNullLayerId() {
         val style = mockk<Style>()
         val layer1 = mockk<Layer>()
```

Comment:
NIT Not a big fan of using digits for naming variables. Could we refactor and give some extra context? Same thing happens with the rest of the tests 👇